fla/ops/log_linear_attn/chunk.py

```
@@ -986,6 +1520,10 @@ def forward(
         if K % BLOCK_K != 0:
             raise ValueError(f"State dimension must be divisible by {BLOCK_K}.")
 
+        if triton.__version__ > "3.2.0":
+            warnings.warn("Triton>3.2.0 detected, which is known to have worse performance. "
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)